web/app.go

```diff
@@ -225,6 +227,8 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 }
 
 func (a *App) Start(ctx context.Context) error {
+	grafana.InitGrafana(ctx, a.config.GrafanaConfig)
```

Comment:
Nothing wrong with this, but, shouldn't we initialize grafana like other dependencies in `DefaultDependencies`?

Reply:
I wanted to pass the ctx to the grafana initialization, we might refactor the init db as well because otherwise the retry blocks any `SIG*`